### ISSUE TITLE
Install without network connectivity

### ISSUE THREAD

**Issue by kg4zow** [mb1986/rm-hacks#323]
My home network is configured to [block my tablets](https://remarkable.jms1.info/info/network.html) from accessing the internet.

**Is there a way to install this *without* having to temporarily un-block the tablets?** Maybe ... I dunno, `git clone` the repo on a computer, `scp -r` that directory into the tablet, and then `ssh` into the tablet and run the installer locally, without it having to access anything outside the local network?

I haven't started looking at what `install.sh` does yet, but if this turns out to be something that *would* work, then all I'm asking for is to update the repo's `README.md` file to explain how to do it.

**Comment by RedTartan04** [comment]
Good idea. With your skills I think you should be able to adapt install.sh a little. From what I see scrolling through, basically it maps the rM's software version to the right patch file and wgets it from github (path in PATCH_URL). In addition the stylus patch option is downloaded from another, hardcoded, url on github (and also updates wget itself). Is it possible to wget a file from a local, relative directory? Then it's maybe just changing the two URLs?
